front/tests/pages/study-page-model.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable import/prefer-default-export */
+import { expect, Locator, Page } from '@playwright/test';
+import study from '../../public/locales/fr/operationalStudies/study.json';
+
+export class StudyPage {
+  // The current page object
+  readonly page: Page;
+
+  // Page informations
+  readonly getStudyName: Locator;
+
+  readonly getStudyDescription: Locator;
+
+  readonly getStudyState: Locator;
+
+  readonly getStudyType: Locator;
+
+  readonly getStudyFinancialsInfos: Locator;
+
+  readonly getStudyFinancialsAmount: Locator;
+
+  readonly getStudyTags: Locator;
+
+  readonly getBackToProject: Locator;
+
+  readonly translation: typeof study;
+
+  readonly getAddStudyBtn: Locator;
+
+  readonly getStudyInputName: Locator;
+
+  readonly getStudyTypeSelect: Locator;
+
+  readonly getStudyStatusSelect: Locator;
+
+  readonly getStudyDescriptionInput: Locator;
+
+  readonly getStudyStartDateInput: Locator;
+
+  readonly getStudyEstimatedEndDateInput: Locator;
+
+  readonly getStudyEndDateInput: Locator;
+
+  readonly getStudyServiceCodeInput: Locator;
+
+  readonly getStudyBusinessCodeInput: Locator;
+
+  readonly getStudyBudgetInput: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    // Initialize locators using roles and text content
+    this.getStudyName = page.locator('.study-details-name .study-name');
```

Comment:
Why do you need study-name and study-details-name ?
```suggestion
    this.getStudyName = page.locator('.study-details-name');
```